docs/data-sources/account_network_policy.md

```diff
@@ -37,9 +37,9 @@ The following attributes are exported:
 ### EgressNetworkPolicyNetworkAccessPolicyStorageDestination
 * `azure_storage_account` (string) - The Azure storage account name
 * `azure_storage_service` (string) - The Azure storage service type (blob, dfs, etc.)
-* `bucket_name` (string) - 
-* `region` (string) - 
+* `bucket_name` (string) -
+* `region` (string) -
```

Comment:
Hi @pietern, thanks for the PR, these are autogenerated docs, so the change would have to be done upstream.